### DISCUSSION TITLE
Show on preview modal size.

### DISCUSSION THREAD

**Discussion by philipbaginski** [laravel/nova-issues#4869] [Ideas & Feature Requests]
I have an resource with picture. It looks fine on the detail page, but using ->showOnPreview() modal is too small to show picture.

Is it possible to make ->showOnPreview() modal with option to adjust size?

**Comment by LTKort** [comment]
Samen here, also the fullWidth options doesn't work on the preview modal. Would be awesome to set the size and fullWidth

**Comment by philipbaginski** [comment]
Please, add modal size for preview modal :-)
